app/src/main/res/layout/view_suggested_edits_task.xml

```diff
@@ -10,55 +10,63 @@
     app:cardBackgroundColor="?attr/paper_color"
     app:cardUseCompatPadding="true">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/task_info_layout"
+    <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:orientation="vertical"
         android:background="?android:attr/selectableItemBackground">
 
-        <TextView
-            android:id="@+id/title"
-            android:layout_width="0dp"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="16dp"
-            android:layout_marginTop="16dp"
-            android:layout_marginEnd="16dp"
-            android:fontFamily="sans-serif-medium"
-            android:lineSpacingExtra="10sp"
-            android:textColor="?attr/material_theme_primary_color"
-            android:textSize="14sp"
-            app:layout_constraintEnd_toStartOf="@id/image"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            tools:text="Add title descriptions" />
-
-        <TextView
-            android:id="@+id/description"
-            android:layout_width="0dp"
+        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
What's the reasoning for changing around these layouts? Is it correct to have `selectableItemBackground` in this layout and its parent?

Reply:
I've noticed in design: https://app.zeplin.io/project/57a120b91998d8977642a238/screen/5c77cfa2b062a1bc6644cc93

The "action layout" design in Zeplin is slightly different than the current one.